src/logic/context/EditorContext.ts

```diff
@@ -0,0 +1,51 @@
+import {HotKeyAction} from "../../data/HotKeyAction";
+import {EditorModel} from "../../model/EditorModel";
+import {LabelType} from "../../data/LabelType";
+import {EditorData} from "../../data/EditorData";
+import {EditorActions} from "../actions/EditorActions";
+import {PolygonRenderEngine} from "../render/PolygonRenderEngine";
+import {EditorSelector} from "../../store/selectors/EditorSelector";
+import {store} from "../../index";
+import {updateActiveImageIndex} from "../../store/editor/actionCreators";
+
+export class EditorContext {
+    public static getActions(): HotKeyAction[] {
+        return [
+            {
+                keyCombo: ["Enter"],
+                action: (event: KeyboardEvent) => {
+                    if (EditorModel.supportRenderingEngine && EditorModel.supportRenderingEngine.labelType === LabelType.POLYGON) {
```

Comment:
tak jak pisałem na messengerze :P te akcje są bardzo niefajne. 
jak mawiał wybitny polski artysta estradowy: "Wiecie co z nim zrobić?"

Reply:
możesz ogarnąć nowe pliki, np: EditorShortcutActions.ts? to może być fajne-  bo ten plik EditorAction będzie przynajmniej mniejszy